source/artwork_import/vectorization.rst

```diff
@@ -1,3 +1,45 @@
-########################
-    Vectorization
-########################
+Vectorization of Bitmaps
+==============================================
+
+Synfig Studio comes with a feature of Vectorization of Bitmaps. In this tutorial, we will learn what is Vectorization feature and how to use it. :)
+
+What is Vectorization of Bitmaps?
```

Comment:
I suggest to change to "What is Vectorization means?"

Reply:
*What does Vectorization mean? 
=)

Reply:
Right =D